app/views/crimes/index.html.erb

```diff
@@ -22,5 +22,8 @@
 </div>
 
 
-
-<%= render partial: "crime" %>
+<div class="container">
```

Comment:
Move the stuff inside this up into the main container div, instead of creating a new container div